src/main/java/com/endava/soa_5/Driver.java

```diff
@@ -33,6 +33,7 @@ private Driver() {
                     throw new RuntimeException("Incorrect browser definition in property file");
             }
         }
+        goToHomePage();
```

Comment:
we don't need this in this specific class

Reply:
done